File: cpp/pst-timestamp.cpp
#include <cstdlib>
#include <ctime>
#include <iostream>

int main() {
    setenv("TZ", "America/Los_Angeles", 1);
    tzset();

    time_t now = time(nullptr);
    struct tm* local = localtime(&now);

    char buffer[64];
    strftime(buffer, sizeof(buffer), "%m/%d/%Y %I:%M %p %Z", local);
    std::cout << buffer << std::endl;
    return 0;
}

File: go/pst-timestamp.go
package main

import (
	"fmt"
	"time"
)

func main() {
	loc, _ := time.LoadLocation("America/Los_Angeles")
	now := time.Now().In(loc)

	// Format: MM/DD/YYYY HH:MM AM/PM MST
	fmt.Println(now.Format("01/02/2006 03:04 PM MST"))
}
